data/src/main/java/com/example/pinterbest/data/models/OwnProfile.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.pinterbest.data.models
+
+import com.example.pinterbest.domain.entities.Profile
+import com.google.gson.annotations.SerializedName
+
+data class OwnProfile(
```

Comment:
By the name of class I should clearly understand what is the layer (data, domain or presentation) the class belongs to?

Reply:
Fixed